docs/UserGuide.md

```diff
@@ -90,7 +90,7 @@ Format: `help`
 
 Adds a student to the student book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]… [c/COMMENT]…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/telegramHandle [t/TAG]… [c/COMMENT]…​`
```

Comment:
Would it be better if you changed this to a/TELEGRAM_HANDLE to stay consistent with all the other fields?